### PR TITLE
feat(hatchery): metrics

### PR DESCRIPTION
**Pull request by fsamin**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: sdk/hatchery/pool.go [hunk 1]
```diff
@@ -1,15 +1,19 @@
 package hatchery
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
 	"github.com/ovh/cds/sdk"
 	"github.com/ovh/cds/sdk/log"
+	"go.opencensus.io/stats"
```
Comment (bnjjj) [inline sdk/hatchery/pool.go]: Bad import

File: engine/api/observability/init.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package observability
 import (
 	"time"
 
+	"github.com/ovh/cds/sdk/log"
```
Comment (bnjjj) [inline engine/api/observability/init.go]: Bad import

File: engine/api/observability/stats.go [hunk 3]
```diff
@@ -5,12 +5,14 @@ import (
 	"net/http"
 
 	"github.com/ovh/cds/engine/service"
+	"github.com/ovh/cds/sdk/log"
 	"go.opencensus.io/stats/view"
```
Comment (bnjjj) [inline engine/api/observability/stats.go]: Bad import

File: engine/hatchery/stats.go [hunk 4]
```diff
@@ -0,0 +1,107 @@
+package hatchery
+
+import (
+	"fmt"
+
+	"github.com/ovh/cds/engine/api/observability"
+
+	"github.com/ovh/cds/sdk/hatchery"
+	"github.com/ovh/cds/sdk/log"
```
Comment (bnjjj) [inline engine/hatchery/stats.go]: Bad import

File: sdk/hatchery/pool.go (hunk 1 above)
Comment (fsamin) [reply sdk/hatchery/pool.go]: happy holiday !

**Comment by fsamin** [comment]
@bnjjj fixed

**Pull request merged**